### DISCUSSION TITLE
Release without tags or Github releases

### DISCUSSION THREAD

**Discussion by lacarvalho91** [goreleaser/goreleaser#2432] [Q&A] [answered]
Hi,

Is it possible to use this tool to just build and publish archives? In a system that is deployed using Continuous Delivery, you may not want to bother with creating tags and creating releases on GitHub.

It would be good if I could use the tool to just build and publish, without any of the GitHub release integration (and therefore not requiring a GitHub token or anything like that)

**Comment by caarlos0** [comment, marked as answer]
maybe the `--snapshot` flag gets you far enough, but I'm not sure as well

**Reply by lacarvalho91** [reply]
I think I tried that, but will try again in the morning and let you know how it goes, thanks!

**Reply by lacarvalho91** [reply]
hmm, that unfortunately seems to prevent publishing from happening. Any way around that?

**Reply by lacarvalho91** [reply]
would you consider the addition of a new flag? something like `--skip-scm-release`?

this would just skip the github/gitlab/gitea release step, thus not requiring any of `GITHUB_TOKEN, GITLAB_TOKEN GITEA_TOKEN` to be set

**Reply by lacarvalho91** [reply]
or a flag to allow snapshot artefacts to be published

**Reply by caarlos0** [reply]
I'm not sure what the side effects of this would be, feel free to investigate though...

**Reply by ranjith-ka** [reply]
Yeha sometimes we work on git flow model, need to publish the artifcat for CD system from release/hotfix/bugfix branches, tag will happen after testing signoff, glad to see the option 👍

**Reply by adasari** [reply]
Just re-opening the discussion. Currently env pipe can't be skipped and forced to use git tags only. is it possible to support publish against non git tag models ? release command doesn't publish artifacts to s3 with `--snapshot` flag.

**Reply by caarlos0** [reply]
without tags no, you can use the nightly feature as a rolling release system if you want though

https://goreleaser.com/customization/nightlies/?h=nightly
